po4a-create-all-targets.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# You need po4a > 0.54, see https://github.com/mquinson/po4a/releases
+# There is no need of system-wide installation of po4a
+# Usage: PERLLIB=/path/to/po4a/lib use_po.sh
+# you may set following variables
+# SRCDIR root of the documentation repository
+# PODIR place where to create the po
+# PUB_DIR place where to publish the localised files
```

Comment:
You should put this directly at the lines where you define these variables

Reply:
This has been done in a previous PR. You seem to be checking the version in "next-release". The one in "release" has those comments in place.

Reply:
Hmm. Ok. Sorry about that. I reviewed this (merged) PR which didn't include the latest changes you did.